src/viur/core/modules/user.py

```diff
@@ -831,6 +836,141 @@ def transaction(user_key, idx):
         db.RunInTransaction(transaction, user_key, idx)
 
 
+class AuthenticatorOTP:
```

Comment:
```suggestion
class AuthenticatorOTP(UserAuthentication):
```
with #800, this has to inherit from `UserAuthentication`